modules/core/src/test/scala/me/mnedokushev/zio/apache/parquet/core/codec/SchemaEncoderDeriverSpec.scala

```diff
@@ -20,7 +20,12 @@ object SchemaEncoderDeriverSpec extends ZIOSpecDefault {
     implicit val schema: Schema[MyEnum] = DeriveSchema.gen[MyEnum]
   }
 
-  case class Record(a: Int, b: Option[String])
+  case class Record(
```

Comment:
For this, I'd say let's create another class (it could be named `GenericRecord`) to test this new functionality separately because now we don't test the "record" case.